### ISSUE TITLE
Unpickling loses activity conditions from Agents

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#195]
Oh boy. Watch what happens to BRAF's activity condition in this one... :-/

```
In [1]: from indra import trips

In [2]: stmts = trips.process_text('Active BRAF phosphorylates MAP2K1 at S218').statements

In [3]: stmts
Out[3]: [Phosphorylation(BRAF(activity: True), MAP2K1(), S, 218)]

In [4]: with open('test.pkl', 'wb') as f:
   ...:     import pickle
   ...:     pickle.dump(stmts, f)
   ...:     

In [6]: with open('test.pkl', 'rb') as f:
   ...:     stmts_pkl = pickle.load(f)
   ...:     

In [7]: stmts_pkl
Out[7]: [Phosphorylation(BRAF(), MAP2K1(), S, 218)]
```
So, remember when we (I) added this to the unpickling step in Statement to stay backwardly compatible with old statement pickle files? Yeah, about that.

https://github.com/sorgerlab/indra/blob/master/indra/statements.py#L473

**Comment by bgyori** [comment]
Oh... just delete that, that was a one-time fix.

**Comment by johnbachman** [comment]
Yes--fixed in f6128ea226256a9761ce496f537be5aadd36ee4c.
